### ISSUE TITLE
Missing midway task

### ISSUE THREAD

**Issue by SBats** [MoonshotProject/moondash#74]
Hi,
the midway task have been removed but there's still a call on it on the default gulp task
`gulp.task('default', ['checker', 'unit', 'midway:watch', 'watch']);`

I don't know if you want to remove completely the task or re add it later, but it's just to let you know.

Have a nice day !

**Comment by pauleveritt** [comment]
> On Apr 16, 2015, at 11:00 AM, Simon Bats <notifications@github.com> wrote:
> 
> Hi,
> the midway task have been removed but there's still a call on it on the default gulp task
> gulp.task('default', ['checker', 'unit', 'midway:watch', 'watch']);
> 
> I don't know if you want to remove completely the task or re add it later, but it's just to let you know.
> 
I think I have it fixed locally on a branch which I haven’t pushed. I have been in the midst of a big refactoring and simplification.

I see you are from Toulouse. How did you hear about Moondash? Interested in giving input on the next round of development?

—Paul

**Comment by SBats** [comment]
I'm actually working in the same company as @ebrehault . We want to try it in production for an app generator, and we noticed with @simon-previdente that the default gulp task couldn't pass.
In fact we'll give to you and @ebrehault every feedback we can :) 
Hopefully it might helps you !

**Comment by pauleveritt** [comment]
> On Apr 16, 2015, at 12:40 PM, Simon Bats <notifications@github.com> wrote:
> 
> I'm actually working in the same company as @ebrehault <https://github.com/ebrehault> . We want to try it in production for an app generator, and we noticed with @simon-previdente <https://github.com/simon-previdente> that the default gulp task couldn't pass.
> In fact we'll give to you and @ebrehault <https://github.com/ebrehault> every feedback we can :) 
> Hopefully it might helps you !
> 
Great! Perhaps in the near future we could setup an IRC session and talk about what to do next.

—Paul

**Comment by SBats** [comment]
Yeah that would be definitely great ! Once we'll be familiar with the full (or at least most of it) potential of Moondash, it would be fun to help you discuss and chose about its future !
